src/adapters/MeetingsJSONAdapter.js

```diff
@@ -854,4 +885,59 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
 
     return concat(initialState$, settingsEvent$);
   }
+
+  /**
+   * Returns an observable that emits the display data of the video options.
+   *
+   * @param {string} ID ID of the meeting for which to update display
+   * @returns {Observable.<MeetingControlDisplay>} Observable that emits control display data
+   * @private
+   */
+  videoControl(ID) {
+    const meeting = this.datasource[ID];
+    const videoDevices = defer(() => this.getDevices('videoinput'));
+    let initialState$;
+
+    if (meeting) {
+      initialState$ = videoDevices.pipe(
+        map((devices) => ({
+          ID: SWITCH_VIDEO_DEVICE,
+          tooltip: 'Video Devices',
+          state: MeetingControlState.INACTIVE,
```

Comment:
I don't think dropdown controls need a state field, it is replaced by the current value, `selected`